tools/flowc/flowc_usage.flow

```diff
@@ -199,6 +199,9 @@ printUsage(config) {
 	prih(help,"      inline-max-nesting=<int>  The boundary for nesting level of primitive calls in functions to be inlined");
 	prih(help,"      do-const-structs=1        Inline variables of limited nesting, which refer to constant structs");
 	prih(help,"      remove-dup-strings=1      Global optimization: remove duplicate string literals with variables, holding common strings");
+	prih(help,"      remove-single-structs=1   Remove structs with only one immutable argument, which are not contained in any union");
```

Comment:
It is not removing, but unfolding or inlining, right?

Reply:
Currently it's completely removing from program.